internal/clients/recommendations/client.go

```diff
@@ -51,11 +40,9 @@ func NewClient(conf RecommendationsClientConfig) (*RecommendationsClient, error)
 	}
 
```

Comment:
In the `NewClient` use the existing *unchanged* configuration to call `NewLazyTokener` to create a new instance. Depending on `offlineToken` either call one or the other `New` function.

Reply:
not sure I understand what youy ask here, in othyer commit you asked to remove the newLazyToken, so I cant use it here